commands/misc.js

```diff
@@ -23,7 +23,7 @@ exports.commands = {
 	randomanswer: function (arg, user, room) {
 		if (room === user) return false;
 		var text = '';
-		var rand = ~~(20 * Math.random());
+		var rand = ~~(23 * Math.random());
```

Comment:
Only updating the spanish language could and will cause errors, should be dynamic anyway (`answers.length`)